### DISCUSSION TITLE
Interact Highlight Ground Items

### DISCUSSION THREAD

**Discussion by Oxycotton** [runelite/runelite#15619] [Ideas]
Mobile literally has this feature, it would be so cool if runelite did =) . When you click a item that is currently on the ground, it gets highlighted.

Pls bring this feature to "interact highlight" plugin <3 <3

https://gyazo.com/f318f5a4fddb70e01cf1cb217cac447e

**Comment by DesertSnak3** [comment]
This is one of the few things I would like to see added to Runelite from the official client(s)
